es-space-mgmt/main/default/lwc/reservationTile/__tests__/reservationTile.test.js

```diff
@@ -0,0 +1,125 @@
+            const paragraphElements = element.shadowRoot.querySelectorAll('p');
+            expect(paragraphElements[0].textContent).toBe(
+                'Customer: ' + RESERVATION_LEAD.record.Lead__r.Name
+            );
+            // check for status text nodes
+            expect(paragraphElements[1].textContent).toBe(
+                'Status: ' + RESERVATION_LEAD.record.Status__c
+            );
+            expect(paragraphElements[2].textContent).toBe(
+                'Market: ' + RESERVATION_LEAD.record.Market__r.Name
+            );
+            expect(paragraphElements[5].textContent).toBe(
+                'Total Guests: ' +
+                    RESERVATION_LEAD.record.Total_Number_of_Guests__c
+            );
+            // check value of formatted date elements
+            const datetimeElements = element.shadowRoot.querySelectorAll(
+                'lightning-formatted-date-time'
+            );
+            expect(datetimeElements[0].value).toBe(
+                RESERVATION_LEAD.record.Start_Date__c
+            );
+            expect(datetimeElements[1].value).toBe(
+                RESERVATION_LEAD.record.End_Date__c
+            );
+        });
+    });
+
+    it('renders reservationTile for Contact', () => {
+        const element = createElement('c-reservation-tile', {
+            is: ReservationTile
+        });
+        element.reservation = RESERVATION_CONTACT;
+        document.body.appendChild(element);
+
+        return Promise.resolve().then(() => {
+            // check for div class
+            const divElement = element.shadowRoot.querySelector('div.pointer');
+            expect(divElement.className).toBe('pointer');
+
+            // check for paragraph text nodes
+            const paragraphElements = element.shadowRoot.querySelectorAll('p');
+            expect(paragraphElements[0].textContent).toBe(
+                'Customer: ' + RESERVATION_CONTACT.record.Contact__r.Name
+            );
+            expect(paragraphElements[1].textContent).toBe(
+                'Status: ' + RESERVATION_CONTACT.record.Status__c
+            );
+            expect(paragraphElements[2].textContent).toBe(
+                'Market: ' + RESERVATION_CONTACT.record.Market__r.Name
+            );
+            expect(paragraphElements[5].textContent).toBe(
+                'Total Guests: ' +
+                    RESERVATION_CONTACT.record.Total_Number_of_Guests__c
+            );
+            // check value of formatted date elements
+            const datetimeElements = element.shadowRoot.querySelectorAll(
+                'lightning-formatted-date-time'
+            );
+            expect(datetimeElements[0].value).toBe(
+                RESERVATION_CONTACT.record.Start_Date__c
+            );
+            expect(datetimeElements[1].value).toBe(
+                RESERVATION_CONTACT.record.End_Date__c
+            );
+        });
+    });
+
+    it('fires a reservationselect', () => {
```

Comment:
You need a duplicate of this test for a lead tile + be more descriptive in your test name "fires a reservationselect when a contact tile is clicked"

Reply:
done!